python_dataset_creator/logging.conf

```diff
@@ -54,4 +54,4 @@ args=(sys.stdout,)
 
 [formatter_simpleFormatter]
 format=%(asctime)s - %(name)s - %(levelname)s - %(message)s
-datefmt=%m/%d/%Y %I:%M:%S %p
\ No newline at end of file
+datefmt=
```

Comment:
was this removed intentionally?

Reply:
Oh I forgot to add it back after debugging. Thanks for catching that!